corgi/api/views.py

```diff
@@ -290,11 +290,15 @@ def taxonomy(self, request, uuid=None):
 class ProductStreamViewSetSet(ProductDataViewSet):
     """View for api/v1/product_streams"""
 
-    queryset = ProductStream.objects.get_queryset()
+    queryset = ProductStream.objects.filter(active=True)
     serializer_class = ProductStreamSerializer
 
     def list(self, request, *args, **kwargs):
         req = self.request
+        active = request.query_params.get("active")
+        if active == "all":
+            self.queryset = ProductStream.objects.filter()
+            return super().list(request)
```

Comment:
I think the return statement should be removed, otherwise ?active=all&ofuri=value won't work together.

Reply:
good catch. Fixed.